README.md

```diff
@@ -1,33 +1,108 @@
+```
+
+Install wireguard on your device, if you have not done this before. For instructions how to do it on your device, please have
+a look on the documentation of your used distribution.
+
+Start the wireguard tunnel.
+
+```bash
+wg-quick up /home/ubunu/ciab.conf
+```
+
+Now your Cloud in a Box is up and usable via the following URLs:
+
+| Name              | URL                                           | Username  | Password                                 |
+|-------------------|-----------------------------------------------|-----------|------------------------------------------|
+| ARA               | <https://ara.services.in-a-box.cloud>         | ara       | password                                 |
+| Flower            | <https://flower.services.in-a-box.cloud>      | -         | -                                        |
+| Grafana           | <https://api.in-a-box.cloud:3000>             | admin     | password                                 |
+| Homer             | <https://homer.services.in-a-box.cloud>       | -         | -                                        |
+| Horizon Admin     | <https://api.in-a-box.cloud>                  | admin     | password                                 |
+| Horizon Test-User | <https://api.in-a-box.cloud>                  | test      | test # FIX ME                            |
+| Kibana            | <https://api.in-a-box.cloud:5601>             | kibana    | password                                 |
+| Netbox            | <https://netbox.services.in-a-box.cloud>      | admin     | password                                 |
+| Netdata           | <http://manager.systems.in-a-box.cloud:19999> | -         | -                                        |
+| phpMyAdmin        | <https://phpmyadmin.services.in-a-box.cloud>  | root      | password                                 |
+| RabbitMQ          | <https://api.in-a-box.cloud:15672>            | openstack | BO6yGAAq9eqA7IKqeBdtAEO7aJuNu4zfbhtnRo8Y |
+
+> **Note:**
+> Watch out for http and https. It is necessary to use the right one.
+
+## Configure Openstack-Client
+
+Connect to your Cloud in a Box via ssh (see command above).
+Select one of the preconfigured environments `system`, `admin`, `test` by exporting the environment variable:
+
+```bash
+export OS_CLOUD=admin
+openstack server list
+```
+
+## Troubleshooting
+
+![Broken disk setup](images/broken_disk_setup.png)
+This error means that your disk setup is broken. Please use `cfdisk` and delete all partitions on the system on which you want
+to install the Cloud in a Box image.
+With `lsblk` you can verify if the partitions are empty.
 
 ## Notes
 
-If you have found a bug, a feature is missing or you have a question just open a
-bug in [osism/issues](https://github.com/osism/issues). We will then move it to
-the right place and assign it as soon as possible.
+If you have found a bug, a feature is missing or you have a question just open a bug in
```

Comment:
just open an issue
("issue" is the wording used in github and also may have a broader meaning than "bug")